src/features/import/l10n/messageIds.ts

```diff
@@ -64,27 +64,23 @@ export default makeMessages('feat.import', {
         value: m('Value'),
       },
       ids: {
-        configExplanation: m(
-          'Importing with IDs allows Zetkin (now or in the future) to update existing people in the database instead of creating duplicates.'
-        ),
         externalID: m('External ID'),
-        externalIDExplanation: m(
-          'The values in this column are IDs from our main member system (not Zetkin).'
+        externalIDInfo: m(
+          'An external ID is an ID that comes from another system than Zetkin - your member system or something of the like. It can be used to find and identify people in Zetkin.'
```

Comment:
```suggestion
          'An external ID is an ID that comes from another system than Zetkin, such as a separate member database. It can be used to find and identify people in Zetkin.'
```